hazelcast/src/test/java/com/hazelcast/sql/impl/exec/scan/index/MapIndexScanExecTest.java

```diff
@@ -405,20 +405,21 @@ private void checkScan(
         List<Integer> projects = Collections.singletonList(0);
 
         MapIndexScanExec exec = new MapIndexScanExec(
-            1,
-            getMapContainer(map),
-            partitions,
-            GenericQueryTargetDescriptor.DEFAULT,
-            GenericQueryTargetDescriptor.DEFAULT,
-            fieldPaths,
-            fieldTypes,
-            projects,
-            remainderFilter,
-            getSerializationService(member),
-            indexName,
-            expectedComponentCount,
-            indexFilter,
-            converterTypes
+                1,
+                getMapContainer(map),
+                partitions,
+                GenericQueryTargetDescriptor.DEFAULT,
+                GenericQueryTargetDescriptor.DEFAULT,
+                fieldPaths,
+                fieldTypes,
+                projects,
+                remainderFilter,
+                getSerializationService(member),
+                indexName,
+                expectedComponentCount,
+                indexFilter,
+                converterTypes,
+                false
```

Comment:
Tests for "descending=true"?

Reply:
@devozerov , I took a look more close at the `MapIndexScanExecTest` and I'm not sure the test is correct. There is a `checkScan` method that is supposed to check in the end the returned results. But instead, it has only the below lines that assert nothing.
```
        results.sort(Integer::compareTo);
        expectedResults0.sort(Integer::compareTo);
```
Moreover, we run the scan only on the local partitions and we actually cannot assume what is the expected result, because the local partitions contain only a part of the entries. 
Can you please clarify this?

Reply:
Perhaps, this is a bug.

Reply:
Generalized the test in https://github.com/hazelcast/hazelcast/pull/18013/commits/07be5e9c4721a297b16c5fb77b4885d925c736f0
Created an issue to fix the tests logic https://github.com/hazelcast/hazelcast/issues/18030